### DISCUSSION TITLE
Can't install watermark

### DISCUSSION THREAD

**Discussion by zhouhao27** [Lightning-AI/dl-fundamentals#110] [General discussions]
When I run `conda install watermark --yes` I got the following error message:

> PackagesNotFoundError: The following packages are not available from current channels:

>  - watermark

Obviously the watermark library is not there. Any hint?

**Comment by rasbt** [comment]
I think that's because you maybe haven't set the conda-forge channel. Can you try

```
conda config --add channels conda-forge
```

and then

```
conda install watermark --yes
```

again?

**Reply by muskmark** [reply]
it worked thanks
